mlflow/store/sqlalchemy_store.py

```diff
@@ -404,13 +405,21 @@ def delete_run(self, run_id):
 
     def log_metric(self, run_id, metric):
         _validate_metric(metric.key, metric.value, metric.timestamp, metric.step)
+        is_nan = math.isnan(metric.value)
+        if is_nan:
+            value = 0
+        elif not math.isfinite(metric.value):
+            #  NB: Sql can not represent Infs = > We replace +/- Inf with max/min 64b float value
+            value = 1.7976931348623157e308 if metric.value > 0 else -1.7976931348623157e308
```

Comment:
Are there Python constants for these values?

Reply:
Yes, but this constant is related to what the database can represent rather than python.

Reply:
As @andrewmchen pointed out, it looks like we define the `value` column as having type `sqlalchemy.Float`. This may be problematic, though I haven't found a double type...

Reference: https://github.com/mlflow/mlflow/blob/849161e0a552b8eabe7634de43b325f5cbda503d/mlflow/store/dbmodels/models.py#L238

Reply:
NIce catch Corey! There is no Double but there appears to be a precision argument so I added a conversion to Float(precision=64) to the migration script.

Reply:
Actually it has to be Float(precision=53) but it works now.